### ISSUE TITLE
Make Polish morphological API usable

### ISSUE THREAD

**Issue by inariksit** [GrammaticalFramework/gf-rgl#427]
Issues:
- ParadigmsPol doesn't export nearly all `mkC` constructors for the common lexical categories; instead, the end user needs to dig into the gf-rgl source code to import internal modules like AdjectiveMorphoPol, NounMorphoPol, MorphoPol
- The opers in those modules, like `mkA`, don't have a nice expected instance like `Str -> A`; instead, they expect as an argument a tuple of 11 fields or something like that.

TODO:
- Populate ParadigmsPol.gf with opers that have the expected names and type signatures

**Comment by chipsf** [comment]
Aside from the API issues you mentioned, there also appear to be some issues with generating grammatically-incorrect phrases, and I think that both problems are coupled. For example,
```
> i LangEng.gfo LangPol.gfo
> p -lang=LangEng "water" | l -lang=LangPol
woda jacy
woda
woda
```
It's not possible for "jacy" to be the determiner associated to "woda". The form "jacy" of the root word "jaki" would agree only with groups of people involving male human beings. We should see "woda jaka" here instead. Anyway, all of that is to say that maybe the Polish grammar needs a refactor. Maybe it could be partially rewritten to look more like the Russian one? 

I'd be willing to assist on this project if you're looking for help on it. I'm interested in working towards becoming a maintainer of this particular grammar, and I have some outside incentives pushing me to do that :)

**Comment by inariksit** [comment]
@chipsf Yes, sounds like a refactor would be useful! The Polish resource grammar is pretty old and we have established nicer conventions than those used at the time it was written. 

And great to hear that you're interested in maintaining the grammar. If you're not yet there, you could come to the GF discord server (https://discord.gg/EvfUsjzmaz) and we can discuss more there!

**Comment by chipsf** [comment]
@inariksit Sounds good. I popped onto the discord server now, so see you there!
